Modules/Sources/Storage/GRDB/Model/PersistedProductAttribute.swift

```diff
@@ -0,0 +1,59 @@
+import Foundation
+import GRDB
+
+public struct PersistedProductAttribute: Codable {
+    public private(set) var id: Int64?
```

Comment:
❓ Any reasons this `autoIncrementedPrimaryKey("id")` field needs to be manually updated in `didInsert` below?